KNM/events/KnightsLanding_uncore.json

```diff
@@ -1332,7 +1332,7 @@
       "EventCode": "0x35",
       "UMask": "0x31",
       "EventName": "UNC_H_TOR_INSERTS.IRQ",
-      "BriefDescription": "Counts the number of entries successfuly inserted into the TOR that match  qualifications specified by the subevent -IRQ",
+      "BriefDescription": "Counts the number of entries successfully inserted into the TOR that match  qualifications specified by the subevent -IRQ",
```

Comment:
Looks good to me. Please increase the version in the header at the top of this file.